ansible/roles/dnsmasq/tasks/main.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install dnsmasq
+  package:
+    name=dnsmasq state=present
+  tags:
+    - setup
+
+- name: Create dnsmasq.d
+  file:
+    path=/etc/dnsmasq.d state=directory owner=root group=root mode=0755
+  tags:
+    - setup
+
+- name: Place dnsmasq.conf
+  template:
+    src=dnsmasq.conf dest=/etc/dnsmasq.conf
+    owner=root group=root mode=0644
+  tags:
+    - setup
```

Comment:
I believe dnsmasq service needs to be restarted (`notify: restart dnsmasq`) if dnsmasq.conf changes, doesn't it?